packages/api/amplify_api_ios/ios/amplify_api_ios.podspec

```diff
@@ -1,11 +1,9 @@
-load 'dependencies.rb'
```

Comment:
Please let me know the migration plan for linting

Reply:
Hmm is this not enough then?

```
s.dependency 'SwiftLint'
s.dependency 'SwiftFormat/CLI'
```

Reply:
No, the script pulls in the actual code that runs the linter. That's what the lines below that got removed did.

Reply:
Alright, looks like I'm missing context here, perhaps you can help.  Looking through other module podspec files such as Auth/Analytics/Datastore etc. I do not see `load 'dependencies.rb'` either ... Does this mean there is no linting for those modules as well?  In this case, it seems like we should be adding `load dependencies.rb` to all module podspec files then right?